apps/judicial-system/web/src/routes/Court/RestrictionCase/HearingArrangements/HearingArrangements.tsx

```diff
@@ -99,6 +99,10 @@ export const HearingArrangements: React.FC = () => {
 
   const handleNavigationTo = useCallback(
     async (destination: keyof stepValidationsType) => {
+      const isCorrectingRuling = workingCase.notifications?.some(
```

Comment:
We treat the case as reopened only if the parties to a case have been notified of a signed ruling. This happens after the ruling is signed by a judge. As a consequence we do not treat a reopened case which ruling has not been signed as being reopened; it is treated as if it was just received.